README.md

```diff
@@ -47,6 +47,14 @@ Additionally, JavaScript and CSS has to be included:
 
 ## Advanced usage
 
+### Hide the smartbanner
+
+If you want to prevent smartbanner rendering in some html pages, you can add optional `meta` tag:
+
+```html
+<meta name="smartbanner:smartbanner-disabled" content="true">
+```
+
```

Comment:
This README section here is the thing I'd keep in this PR to make use of it. I'd move it to be the last section in __Advanced usage__ and rename to __Hide the smartbanner completely__.